torch/testing/_internal/common_utils.py

```diff
@@ -942,16 +942,16 @@ def __torch_function__(self, func, types, args=(), kwargs=None):
         return r
 
 # Run PyTorch tests with TorchDynamo
-TEST_WITH_TORCHINDUCTOR = os.getenv('PYTORCH_TEST_WITH_INDUCTOR') == '1'
-TEST_WITH_TORCHDYNAMO = os.getenv('PYTORCH_TEST_WITH_DYNAMO') == '1' or TEST_WITH_TORCHINDUCTOR
+# This used to be set with the flags below, but it is disabled for now
+# os.getenv('PYTORCH_TEST_WITH_INDUCTOR') == '1'
+# os.getenv('PYTORCH_TEST_WITH_DYNAMO') == '1' or TEST_WITH_TORCHINDUCTOR
+TEST_WITH_TORCHINDUCTOR = False
+TEST_WITH_TORCHDYNAMO = False
```

Comment:
Why do this? You disabled the envvar in test.sh, so shouldn't you just keep this code here?